yaptide/batch/batch_methods.py

```diff
@@ -309,19 +309,21 @@ def get_job_results(simulation: BatchSimulationModel, user: KeycloakUserModel, c
     return {"message": "Results not available"}
 
 
-def delete_job(simulation: BatchSimulationModel, user: KeycloakUserModel,
-               cluster: ClusterModel) -> tuple[dict, int]:  # skipcq: PYL-W0613
+def delete_job(simulation: BatchSimulationModel, user: KeycloakUserModel, cluster: ClusterModel,
+               to_fetch: bool) -> tuple[dict, int]:  # skipcq: PYL-W0613
     """Dummy version of delete_job"""
     job_dir = simulation.job_dir
     array_id = simulation.array_id
     collect_id = simulation.collect_id
 
     try:
         con = get_connection(user=user, cluster=cluster)
-
-        con.run(f'scancel {array_id}')
-        con.run(f'scancel {collect_id}')
-        con.run(f'rm -rf {job_dir}')
+        if to_fetch:
+            con.run(f'scancel --signal SIGINT {array_id}')
```

Comment:
What about killing merge job if we fetch results?

Reply:
We are in `delete_job` function, I would expect it terminates the job completely.
Maybe for better clarity you should define a separate function which would be named `fetch_results_without_stopping` or something that bears some meaning ?